x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_notify_when.tsx

```diff
@@ -77,7 +77,7 @@ export const NOTIFY_WHEN_OPTIONS: Array<EuiSuperSelectOption<RuleNotifyWhenType>
         <EuiText size="s" color="subdued">
           <p>
             <FormattedMessage
-              defaultMessage="Actions repeat at the rule interval when the alert is active."
+              defaultMessage="Actions repeat at each check interval."
```

Comment:
Alternatively, could have this option as:

**Every check interval**
Actions repeat if the alert is active.

Reply:
A wordier explanation if this helps: `Actions run when alert becomes active and repeat at each check interval if alert remains active`.

Reply:
Thanks, I've updated the docs and the UI text to reflect that info.